.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+.ansible
```

Comment:
Seems like the file has an invalid EOF or at least a missing "newline" at the end of the last line.
```suggestion
.ansible
```

Reply:
.gitignore is not in yaml format. There should be a newline needed.